system/mount.py

```diff
@@ -124,6 +126,8 @@ def set_mount(module, **kwargs):
     origname =  args['name']
     # replace any space in mount name with '\040' to make it fstab compatible (man fstab)
     args['name'] = args['name'].replace(' ', r'\040')
+    # remove trailing slash in mount name (if exists)
+    args['name'] = re.sub('/$','',args['name'])
```

Comment:
Hi @peerster!

Thanks for the feedback. You're absolutely right, rstrip is a better method.

But - I'm kinda new at github...
Should I edit the code or create a new commit and then a new pull request?